lib/ts/recipe/session/types.ts

```diff
@@ -146,19 +145,19 @@ export interface VerifySessionOptions {
 
 export interface RecipeInterface {
```

Comment:
These should all be `any` instead of `BaseResponse` or `BaseRequest`

Reply:
done